src/main/java/io/cdap/plugin/common/SFTPActionConfig.java

```diff
@@ -43,8 +43,26 @@
   @Macro
   public String userName;
 
+  @Description("Private Key to be used to login to SFTP Server. SSH key must be of RSA type")
+  @Macro
+  @Nullable
+  public String privateKey;
+
+  @Description("Passphrase to be used with private key if passphrase was enabled when key was created. " +
+    "If PrivateKey is selected for Authentication")
+  @Macro
+  @Nullable
+  public String passphrase;
```

Comment:
What if this is not provided? would SFTP connection still work when this is null?

Reply:
yes, it will be null in the connector object which would specify that the RSA key was created without a passphrase when the library verifies the private and public SSH keys